src/main/java/io/wispforest/owo/command/debug/HealCommand.java

```diff
@@ -19,14 +20,16 @@ public class HealCommand {
 
     public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
         dispatcher.register(literal("heal")
+                .executes(context -> execute(context, context.getSource().getPlayerOrThrow(), context.getSource().getPlayerOrThrow().getMaxHealth()))
+                .then(argument("amount", FloatArgumentType.floatArg(0))
+                        .executes(context -> execute(context, context.getSource().getPlayerOrThrow(), FloatArgumentType.getFloat(context, "amount"))))
```

Comment:
Same as with damage, make these methods with descriptive names which call successively more generic methods that actually contain logic